src/Tribe/Main.php

```diff
@@ -4355,8 +4355,22 @@ public function setup_date_search_in_bar( $filters ) {
 
 			global $wp_query;
 
+			/**
+			 * Allows for customizing the "date search" field value.
+			 *
+			 * @deprecated TBD Use tribe_events_bar_date_search_default_value instead.
+			 *
+			 * @param string $value The "date search" field value, which defaults to an empty string.
+			 */
 			$value = apply_filters( 'tribe-events-bar-date-search-default-value', '' );
 
+			/**
+			 * Allows for customizing the "date search" field value.
+			 *
+			 * @param string $value The "date search" field value, which defaults to an empty string.
+			 */
+			$value = apply_filters( 'tribe_events_bar_date_search_default_value', '' );
```

Comment:
If a 3rd party is using the former and now deprecated hook, we should still try to support them and avoid breakages. To that end, let's pass `$value` here instead of overwriting it by passing empty string `''`